apps/bare-expo/app.json

```diff
@@ -44,6 +44,11 @@
     "runtimeVersion": {
       "policy": "appVersion"
     },
+    "updates": {
+      "checkAutomatically": "NEVER",
+      // Workaround to use expo/expo URL since we don't have an EAS update project for bare-expo yet.
+      "url": "https://github.com/expo/expo"
```

Comment:
It looks like there's an EAS project ID above? `2c28de10-a2cd-11e6-b8ce-59d1587e6774`

Reply:
https://expo.dev/accounts/expo/projects/native-component-list

Reply:
i thought it's ncl not bare-expo, but looks like it works for using ncl for bare-expo. thanks will update accordingly.